homeassistant/helpers/service.py

```diff
@@ -29,12 +31,21 @@ def register_service_decorator(action):
 
 def call_from_config(hass, config, blocking=False):
     """Call a service based on a config hash."""
-    if not isinstance(config, dict) or CONF_SERVICE not in config:
+    if not isinstance(config, dict):
+        _LOGGER.error('Invalid configuration %s', config)
+        return
+
+    if CONF_SERVICE not in config and CONF_SERVICE_TEMPLATE not in config:
         _LOGGER.error('Missing key %s: %s', CONF_SERVICE, config)
         return
 
+    domain_service = (
```

Comment:
'service' is already defined. Could not come up with a better name than 'domain_service'